tokio/src/runtime/scheduler/multi_thread/queue.rs

```diff
@@ -118,8 +127,44 @@ impl<T> Local<T> {
         !self.inner.is_empty()
     }
 
+    pub(crate) fn push_back(&mut self, task: task::Notified<T>) -> Result<(), task::Notified<T>> {
+        let tail = loop {
```

Comment:
The `loop` here probably can be removed, since the body currently is always exited in the first iteration, either via `break` or `return`.

Reply:
Done